src/zenml/utils/docker_utils.py

```diff
@@ -244,6 +244,7 @@ def build_docker_image(
                 base_image=base_image or DEFAULT_BASE_IMAGE,
                 entrypoint=entrypoint,
                 requirements=requirements,
+                environment_vars=environment_vars,
```

Comment:
This for sure is neccessary but I am wondering if there is a way to `obfuscate` these env vars which are secrets in the logs. Your screenshot shows they are printed out on the console which might not be desirable fo some secrets.

Reply:
Yeah I would think we don't want to have secrets visible in logs.

Reply:
I think the logging part is a built-in thing in docker and am not sure we could hide or change how this secret looks on it! tho I think if we want to add secrets without having them visible in docker inspect or with docker exec. we need to create a script that can be executed to register all the environments.

Reply:
Maybe a stretch but maybe we can use docker secret manager? https://docs.docker.com/engine/swarm/secrets/ . It feels like docker has this ability to write to `/var/www/secrets` or something. But maybe its overkill for now... If its not possible easily id say we let it go but add a TODO [HIGH] for this